carbon-design-system/carbon
# fix: showPasswordLabel and hidePasswordLabel not working

Closes #15986 

ShowPasswordLabel and HidePasswordLabel props were not working.

#### Testing / Reviewing

1. Create a PasswordInput component 
2. hover on eye icon on right side of password input field.
3. It should display text we assign to prop 'showPasswordLabel'
4. Now click on that eye icon.
5. on hover it should display text we assign to prop 'hidePasswordLabel'
